.formatter.exs

```diff
@@ -0,0 +1,3 @@
+[
+  inputs: ["{mix,.formatter}.exs", "{core,eal,lib,local,test}/**/*.{ex,exs}"]
```

Comment:
This format has too many line breaks for me. I prefer larger `:line_length`.

Reply:
Do you have a recommended value as a maximum line length?

Reply:
100 or 120 ? I think it is too large if it is larger than 120.

Reply:
The default value of `:line_length` is [98](https://github.com/elixir-lang/elixir/blob/4fea9cc942f632d5a1692eb85774d097546bc4e5/lib/elixir/lib/code.ex#L653) which elixir style guide [recommends](https://github.com/christopheradams/elixir_style_guide#whitespace).

IMO, the current line length of Antikythera is too long for me who uses 12-13inch MacBook with vertically splitted editor.

I'd like to use 98.